test/qweb3.js

```diff
@@ -511,5 +512,23 @@ describe('Qweb3', () => {
         assert.isDefined(res);
       });
     });
+
+    describe('backupWallet()', () => {
+      it('backup the wallet', async () => {
+        const res = await qweb3.backupWallet(path.join(__dirname, './data/backup.dat'));
+        assert.notTypeOf(res, 'Error');
+      });
+    });
+
+    describe('importWallet()', () => {
+      it('imports keys from a wallet dump file', async () => {
```

Comment:
lets have another test, so 2 tests for importWallet:
1. it imports keys from a wallet dump file 
2. it throws an error when trying to import an invalid file (this one is already done)

When writing tests, you have to consider:
- what are all the possible cases
- what are all the possible outcomes
- what values are being changed/returned
- etc

Reply:
@dwalintukan for `importwallet`, we have 3 scenarios. 1: import valid file 2. import invalid file 3. import non-exist file. 
for 1 and 2, the responses are the same:  
```
{
    "result": null,
    "error": null,
    "id": null
}
```
for 3:
```
{
    "result": null,
    "error": {
        "code": -8,
        "message": "Cannot open wallet dump file"
    },
    "id": null
}
```

so what we have done is for scenario 3, we use `try` `catch` for it. but for scenario 1 and 2, we can not differentiate them.

Reply:
That's ok. Let's cover number 1 with a valid wallet.dat file. Don't worry about number 2 since Qtum doesn't even validate that either.

Reply:
This test name is the wrong case. It should be something like it(`throws an error if importing a non-existent file`)

Reply:
Add the success case: `it imports the wallet from a wallet dump file`